src/core/value.rs

```diff
@@ -1,12 +1,47 @@
+// Rust Simplicity Library
+// Written in 2020 by
+//   Andrew Poelstra <apoelstra@blockstream.com>
+//   Sanket Kanjalkar <sanket1729@gmail.com>
+//
+// To the extent possible under law, the author(s) have dedicated all
+// copyright and related and neighboring rights to this software to
+// the public domain worldwide. This software is distributed without
+// any warranty.
+//
+// You should have received a copy of the CC0 Public Domain Dedication
+// along with this software.
+// If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+//
+
+//! # Simplicity values
+//!
+//! Simplicity processes data in terms of [`Value`]s,
+//! i.e., inputs, intermediate results and outputs.
+//!
+//! Rust values can be converted to Simplicity values and back.
```

Comment:
In 024f9115bc3f626199ebc464c6cf507bf96449fd:

I'd drop this line. It's not true in general.